src/test/java/gov/usgs/wma/waterdata/collections/ObservationsStatTimeSeriesControllerIT.java

```diff
@@ -0,0 +1,84 @@
+package gov.usgs.wma.waterdata.collections;
+
+import static org.hamcrest.CoreMatchers.*;
+import static org.hamcrest.MatcherAssert.*;
+import static org.junit.jupiter.api.Assertions.*;
+import static uk.co.datumedge.hamcrest.json.SameJSONAs.*;
+import static gov.usgs.wma.waterdata.collections.ObservationsStatTimeSeriesController.*;
+
+import org.json.JSONObject;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import com.github.springtestdbunit.annotation.DatabaseSetup;
+
+@SpringBootTest(webEnvironment=WebEnvironment.RANDOM_PORT)
+@DatabaseSetup("classpath:/testData/monitoringLocation/")
+@DatabaseSetup("classpath:/testData/groundwaterDailyValue/")
+public class ObservationsStatTimeSeriesControllerIT extends BaseCollectionsIT {
+	
+	
+	String makeURL(String collectionId, String featureId, String guid) {
+		String url = URL_STATISTICAL_TIME_SERIES.replace(URL_PARAM_COLLECTION_ID, collectionId);
+		url = url.replace(URL_PARAM_FEATURE_ID, featureId);
+		url = url.replace(URL_PARAM_TIME_SERIES_ID, guid);
+		url = "/" + url.replaceAll("[{}]", "");
+		System.out.println(url);
+		return url;
+	}
+
+	@Test
+	public void foundTimeSeriesTest() throws Exception {
+		String url = makeURL(DEFAULT_COLLECTION_ID, "USGS-07227448", "e6a4cc2de5bf437e83efe0107cf026ac");
+//		String url = "/collections/monitoring-locations/items/USGS-07227448/time-series/e6a4cc2de5bf437e83efe0107cf026ac";
```

Comment:
more nitpicky. If line 36 is left in for documentation and never expected to ever be uncomment out, make it
read more like a comment: url returned is: ...